### DISCUSSION TITLE
Update translation source files

### DISCUSSION THREAD

**Discussion by JulianGro** [WeblateOrg/weblate#7361] [Q&A] [answered]
How would one go about automatically updating translation files?

In my case we are using Sphinx for our documentation and need Sphinx to add new strings or changes to the PO files that end up in Weblate.
On our self hosted instance I just used Git hooks to have Weblate run the specific Sphinx commands, however I don't think that be done on Hosted Weblate.
Is it possible for (Hosted) Weblate to run through a little "build" process on ever repository change?
Or would we need to do that ourselves outside of Weblate?

**Comment by nijel** [comment, marked as answer]
Currently, it is not possible. It would require some kind of workers to securely execute untrusted code, and I believe it is better to use whatever CI you already have for that instead of building a similar service inside Weblate.
